recipe/meta.yaml

```diff
@@ -17,19 +17,19 @@ requirements:
   build:
     - cmake
     - python
```

Comment:
@msarahan I'm playing with this recipe and the new conda-build 3 logic.
If I understand things correctly I can everything but `cmake` to the `host` section, right?
Also, I could eliminate the `run` entirely, correct?

Sorry for such a dummy question. I'll get that eventually :smile: